src/main/java/bio/terra/common/DaoUtils.java

```diff
@@ -139,4 +140,16 @@ public static Object stringToProperties(ObjectMapper objectMapper, String proper
       return null;
     }
   }
+
+  public static class UuidMapper implements RowMapper<UUID> {
+    private final String columnLabel;
+
+    public UuidMapper(String columnLabel) {
+      this.columnLabel = columnLabel;
+    }
+
+    public UUID mapRow(ResultSet rs, int rowNum) throws SQLException {
```

Comment:
This should have an `@Override`. If you don't see a warning for this in IntelliJ, you can enable it by going to `Inspections / Java / Inheritance issues / Missing @Override annotation`.
```suggestion
    @Override
    public UUID mapRow(ResultSet rs, int rowNum) throws SQLException {
```

Reply:
I did not see a warning for this, but I do have this setting enabled.  I'll have to debug further but can add this in.